### PR TITLE
Docs: Change checkout to check out where necessary

### LINKED ISSUE #20775: Docs: Change checkout to check out
Review the documentation and change "checkout" to "check out" where appropriate.

"Checkout" is a noun. Example: Proceed to checkout.
"Check out" is a verb. Example: Check out these new features.

In many places in the documentation, the noun version is used where the verb should be. 

Note: Please comment on this issue when you start working on it so we do not get more than one person doing the same thing.

### PR DESCRIPTION
**Pull request by sebimarkgraf**
Change checkout in documentation to check out where necessary.

Limited to files in documentation to not create merge conflicts.

Fixes #20775

### CODE REVIEW

File: docs/sources/features/datasources/opentsdb.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Query | Description
 *tag_values(cpu, hostname, env=$env)*  | Return tag values for cpu metric, selected env tag value and tag key hostname
 *tag_values(cpu, hostname, env=$env, region=$region)* | Return tag values for cpu metric, selected env tag value, selected region tag value and tag key hostname
 
-For details on OpenTSDB metric queries checkout the official [OpenTSDB documentation](http://opentsdb.net/docs/build/html/index.html)
+For details on OpenTSDB metric queries check out the official [OpenTSDB documentation](http://opentsdb.net/docs/build/html/index.html)
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/opentsdb.md]:
```suggestion
For details on OpenTSDB metric queries, check out the official [OpenTSDB documentation](http://opentsdb.net/docs/build/html/index.html)
```

File: docs/sources/plugins/installation.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Grafana already have a strong community of contributors and plugin developers.
 By making it easier to develop and install plugins we hope that the community
 can grow even stronger and develop new plugins that we would never think about.
 
-To discover plugins checkout the official [Plugin Repository](https://grafana.com/plugins).
+To discover plugins check out the official [Plugin Repository](https://grafana.com/plugins).
```
Comment (oddlittlebird) [inline docs/sources/plugins/installation.md]:
```suggestion
To discover plugins, check out the official [Plugin Repository](https://grafana.com/plugins).
```

**Comment by oddlittlebird** [comment]
Thanks for doing this! Looks great, just a couple of minor suggestions to apply since you're in the content.

**Comment by Sintifo** [comment]
@oddlittlebird Applied your suggestions.

**Review by oddlittlebird — COMMENTED** [review]
LGTM!

**Review by oddlittlebird — APPROVED** [review]
Approved

**Pull request merged**